tests/conftest.py

```diff
@@ -121,16 +193,127 @@ def hardened(request):
 @pytest.fixture
 def _file(request):
     """Use a local file directory for root-dir instead of FTP or HTTP."""
-    return request.config.getoption("--file")
+    return request.config.getoption('--file')
 
 
 @pytest.fixture
 def auth(request):
     """Authentication method for HTTP fetching. Valid values: basic, digest"""
-    return request.config.getoption("--auth")
+    return request.config.getoption('--auth')
 
 
 @pytest.fixture
 def noupdate(request):
     """ Decide whether or not to update the fetch to the latest patch level."""
-    return request.config.getoption("--noupdate")
+    return request.config.getoption('--noupdate')
+
+
+@pytest.fixture(scope='session')
+def jails():
+    return {}
+
+
+@pytest.fixture
+def config_data():
+    def read_file(jail):
+        jails = iocage.IOCage(jail=jail).jails
```

Comment:
Let's skip all of this, as that will have a significant performance cost. Let's do this:
```suggestion
    return iocage_lib.ioc_json.IOCJson(path).json_get_value('all')
```

Reply:
Redoing this in `data_classes` differently ;)